server/eclipse-project/src/main/java/us/freeandfair/corla/controller/ComparisonAuditController.java

```diff
@@ -315,10 +315,13 @@ public static boolean submitAuditCVR(final CountyDashboard the_dashboard,
     }
 
     updateCVRUnderAudit(the_dashboard);
+    final Round current_round = the_dashboard.currentRound();
```

Comment:
Why did we move this here?

Reply:
It made more sense to update it here than to update it in `updateCVRUnderAudit()`; the effect is the same.